v/voary.md

```diff
@@ -0,0 +1,17 @@
+# Artist name
+
+![voary](voary.jpg)
+
+Feb 18, 2016 - Uploaded by Les réalisations de Aina Ralai
+Hajaniaina RALAIMAMPIANINA a dirigé la réalisation de ce clip vidéo "Tsy ho rava", nous remercions PARH-SHON ...
+
+May 29, 2014 - Uploaded by Sheima Clarence
+Sheima Clarence. ... 50+ videos Play all Mix - AMPY IZAY: Parh-Shon sy VoaryYouTube. ... Sheima Clarence ...
+You visited this page on 1/28/20.
+
+This artist started in ... and became famous ...
+
+**Genre:** Slow
+
+**Artist homepage:** [Google.com](www.youtube.com)
```

Comment:
Please try find a homepage for Voary. If you can't find one, then link to their Wikipedia page, or Last.fm, or Discog page.